### PR TITLE
add extension to relative imports to make the code work directly in a browser

### PR DESCRIPTION
**Pull request by JakeChampion**
relative imports require the full path including the extension because browsers use the import name to make a network request and they do not implicitly add a .js file extension if one is missing

### CODE REVIEW

File: src/js/Sort/TableSorter.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import CellFormatter from "./CellFormatter";
+import CellFormatter from "./CellFormatter.js.js";
```
Comment (JakeChampion) [inline src/js/Sort/TableSorter.js:1]:
```suggestion
import CellFormatter from "./CellFormatter.js";
```

File: src/js/Tables/BasicTable.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import BaseTable from './BaseTable';
+import BaseTable from './BaseTable.js.js';
```
Comment (JakeChampion) [inline src/js/Tables/BasicTable.js:1]:
```suggestion
import BaseTable from './BaseTable.js';
```

File: src/js/Tables/FlatTable.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import BaseTable from './BaseTable';
+import BaseTable from './BaseTable.js.js';
```
Comment (JakeChampion) [inline src/js/Tables/FlatTable.js:1]:
```suggestion
import BaseTable from './BaseTable.js';
```

File: src/js/Tables/OverflowTable.js [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import BaseTable from './BaseTable';
+import BaseTable from './BaseTable.js.js';
```
Comment (JakeChampion) [inline src/js/Tables/OverflowTable.js:1]:
```suggestion
import BaseTable from './BaseTable.js';
```

File: src/js/Tables/ScrollTable.js [hunk 5]
```diff
@@ -1,4 +1,4 @@
-import BaseTable from './BaseTable';
+import BaseTable from './BaseTable.js.js';
```
Comment (JakeChampion) [inline src/js/Tables/ScrollTable.js:1]:
```suggestion
import BaseTable from './BaseTable.js';
```

File: src/js/oTable.js [hunk 6]
```diff
@@ -1,9 +1,9 @@
-import FlatTable from './Tables/FlatTable';
-import ScrollTable from './Tables/ScrollTable';
-import OverflowTable from './Tables/OverflowTable';
-import BasicTable from './Tables/BasicTable';
+import FlatTable from './Tables/FlatTable.js.js';
+import ScrollTable from './Tables/ScrollTable.js.js';
+import OverflowTable from './Tables/OverflowTable.js.js';
+import BasicTable from './Tables/BasicTable.js.js';
```
Comment (JakeChampion) [inline src/js/oTable.js:4]:
```suggestion
import FlatTable from './Tables/FlatTable.js';
import ScrollTable from './Tables/ScrollTable.js';
import OverflowTable from './Tables/OverflowTable.js';
import BasicTable from './Tables/BasicTable.js';
```

File: src/js/oTable.js [hunk 7]
```diff
@@ -1,9 +1,9 @@
...
 
-import TableSorter from './Sort/TableSorter';
+import TableSorter from './Sort/TableSorter.js.js';
```
Comment (JakeChampion) [inline src/js/oTable.js:6]:
```suggestion
import TableSorter from './Sort/TableSorter.js';
```

**Comment by origamiserviceuser** [comment]
o-table bundle size difference from 8.1.3 to 5eb94de94bf5ebf9327e9d9f49cd4d6345f34b24
No significant bundle size differences found.

**Review by chee — APPROVED** [review]

**Review by chee — APPROVED** [review]

**Comment by origamiserviceuser** [comment]
o-table bundle size difference from 8.1.3 to 361ab7218cf41d0f11899dffa112232ba5034ecc
No significant bundle size differences found.

**Comment by origamiserviceuser** [comment]
:tada: This PR is included in version v8.1.4 :tada:

**Pull request merged**